src/main/java/com/minecolonies/coremod/colony/requestsystem/requests/StandardRequestFactories.java

```diff
@@ -0,0 +1,283 @@
+import com.minecolonies.api.colony.requestsystem.request.IRequestFactory;
+import com.minecolonies.api.colony.requestsystem.requestable.Delivery;
+import com.minecolonies.api.colony.requestsystem.requester.IRequester;
+import com.minecolonies.api.colony.requestsystem.token.IToken;
+import com.minecolonies.api.util.constant.Suppression;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.nbt.NBTTagInt;
+import net.minecraft.nbt.NBTTagList;
+import net.minecraftforge.common.util.Constants;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Final class holding all the factories for requestables inside minecolonie
+ */
+public final class StandardRequestFactories
+{
+
+    ////// --------------------------- NBTConstants --------------------------- \\\\\\
+    private static final String NBT_TOKEN     = "Token";
+    private static final String NBT_STATE     = "State";
+    private static final String NBT_REQUESTED = "Requested";
+    private static final String NBT_RESULT    = "Result";
+    private static final String NBT_PARENT    = "Parent";
+    private static final String NBT_CHILDREN  = "Children";
+    ////// --------------------------- NBTConstants --------------------------- \\\\\\
+
+    /**
+     * Private constructor to hide the implicit public one.
+     */
+    private StandardRequestFactories()
+    {
+    }
+
+    @SuppressWarnings(Suppression.BIG_CLASS)
+    public static final class ItemStackFactory implements IRequestFactory<ItemStack, StandardRequests.ItemStackRequest>
+    {
+        /**
+         * Method to get a new instance of a request given the input and token.
+         *
+         * @param input        The input to build a new request for.
+         * @param location     The location of the requester.
+         * @param token        The token to build the request from.
+         * @param initialState The initial state of the request request.
+         * @return The new output instance for a given input.
+         */
+        @Override
+        public StandardRequests.ItemStackRequest getNewInstance(
+                                                                 @NotNull final ItemStack input,
+                                                                 @NotNull final IRequester location,
+                                                                 @NotNull final IToken token,
+                                                                 @NotNull final RequestState initialState)
+        {
+            return new StandardRequests.ItemStackRequest(location, token, initialState, input);
+        }
+
+        @NotNull
+        @Override
+        @SuppressWarnings(Suppression.LEFT_CURLY_BRACE)
+        public TypeToken<StandardRequests.ItemStackRequest> getFactoryOutputType()
+        {
+            return new TypeToken<StandardRequests.ItemStackRequest>() {};
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)